### DISCUSSION TITLE
Supabase Performance

### DISCUSSION THREAD

**Discussion by 2020-byte** [supabase/supabase#28965] [Questions]
I've tested the speed of my website, which is using Supabase with a region set to Canada, to operate in Japan as well. On the server side, I'm using the following two calls:

const {
  data: { user },
} = await supabase.auth.getUser();

const { data: profile } = await supabase
  .from("profiles")
  .select()
  .eq("id", user?.id)
  .single();
  
It takes about 1 second for the page to render. Is this an expected speed, or is it considered too slow?

**Comment by GaryAustin1** [comment]
You don't really have much info on what part of that is the supabase call time/latency.

I typically see 150 to 250 msec for the Option and Data request to US-east from Texas measured in the browser network tab for a database select.  The SQL part of that is sub 1msec.
